### DISCUSSION TITLE
Spoolman Temperature Override

### DISCUSSION THREAD

**Discussion by Axaroz** [Donkie/Spoolman#157] [Q&A] [answered]
Hi!

I am really enjoying spoolman and it works great together with moonraker/mainsail.
But unable to get Temperature override to work, is there anything special that needs to be enabled or added to the config for it to work? thanks!

**Comment by Donkie** [comment, marked as answer]
Hi, glad you like it :)
Currently, the temperature fields are only informative and moonraker/klipper does not make use of them in any way to change your printer behaviour. This is something that I'm planning to look into however if it's possible to make work.
